src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -91,8 +92,33 @@ public static function init_default_lists() {
 					'Appearance',
 				),
 				'event_prefix' => 'tasklist_',
+				'visible'      => ! Features::is_enabled( 'tasklist-setup-experiment-1' ),
 			)
 		);
+
+		self::add_list(
+			array(
+				'id'           => 'setup_experiment_1',
+				'hidden_id'    => 'setup',
+				'title'        => __( 'Get ready to start selling', 'woocommerce-admin' ),
+				'tasks'        => array(
+					'StoreDetails',
+					'Products',
+					'WooCommercePayments',
+					'Payments',
+					'Tax',
+					'Shipping',
+					'Marketing',
+					'Appearance',
+				),
+				'event_prefix' => 'tasklist_',
+				'options'      => array(
+					'task_past_tense' => true,
```

Comment:
I have the feeling that `task_past_tense` is not very intuitive, could we use something like `completed_title` or `use_completed_title` instead?

Reply:
Thanks, I like `use_completed_title`